docs/features/readonly-variables-and-parameters.md

```diff
@@ -0,0 +1,61 @@
+### High Level Language Proposal:  
+Allow 'readonly' to be used on local variables and on parameters. 
+
```

Comment:
I agree with Cyrus here, because `const` is transitively immutable, while `readonly` can allow mutation outside of the reference.